apps/front/static/img/sponsors/logo_coredump.svg

```diff
@@ -0,0 +1,84 @@
+       style="fill:#000000" />
+    <text
+       id="text4913-9-7-9-5"
+       y="311.22037"
+       x="338.16034"
+       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:20px;line-height:125%;font-family:OCRB;-inkscape-font-specification:'OCRB, Normal';text-align:center;letter-spacing:0px;word-spacing:0px;writing-mode:lr-tb;text-anchor:middle;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
+       xml:space="preserve"><tspan
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:20px;line-height:125%;font-family:OCRB;-inkscape-font-specification:'OCRB, Normal';text-align:center;writing-mode:lr-tb;text-anchor:middle"
+         y="311.22037"
+         x="338.16034"
+         id="tspan4915-2-6-1-7">coredump</tspan></text>
+    <text
+       id="text4913-9-7-9-5-7"
+       y="330.22513"
+       x="342.49536"
+       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:20px;line-height:125%;font-family:OCRB;-inkscape-font-specification:'OCRB, Normal';text-align:center;letter-spacing:0px;word-spacing:0px;writing-mode:lr-tb;text-anchor:middle;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
+       xml:space="preserve"><tspan
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:125%;font-family:OCRB;-inkscape-font-specification:'OCRB, Normal';text-align:center;writing-mode:lr-tb;text-anchor:middle"
+         y="330.22513"
+         x="342.49536"
+         id="tspan4915-2-6-1-7-8">Rapperswil CH</tspan></text>
```

Comment:
Haben wir inzwischen nicht eine Logo-Variante, auf der "Rapperswil-Jona CH" anstatt nur "Rapperswil CH" steht? Da das Vinora-Areal (derzeitiger Standort des Hacker- und Makerspaces) in Jona liegt, würde ich diese Logo-Variante mit "Jona" in der Ortsangabe verwenden.

Reply:
Bisher nicht, ich habe nach dem Umzug mal kurz versucht den Text auf "Rapperswi-Jona" zu ändern. Hab's aber auf die Schnelle nicht geschafft, es hinzukriegen dass es nicht crappy aussieht. Nur Jona wäre natürlich auch eine Option.

Du bringst mich aber auf etwas anderes: Das Logo ist nicht in Kurven umgewandelt und sieht deshalb nicht korrekt aus wenn die Font fehlt.

Reply:
Ist updated. Sieht ok-ish aus. (Ich wäre ja nach wie vor dafür, einen Grafiker mit Logo-Gestaltung zu beaufragen statt einen Informatiker :stuck_out_tongue:)

Reply:
Hab' grad nachgeguckt: Auf den Coredump-T-Shirts haben wir "Rapperswil-Jona CH" unter dem Logo stehen und crappy sieht's meines Erachtens nicht aus.